editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -180,6 +178,23 @@ function Component(props: Props) {
   }
 }
 
+const AutocompleteWrapper = styled(Box)(({ theme }) => ({
```

Comment:
Something worth noting with Mui v5 styles - this needed to be hoisted outside of `GetAddress` for it to work and render correctly.